common/containers/Tabs/ENS/components/NameInput/components/HOC.jsx

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
+import { isValidENSName } from 'libs/validators';
+type State = {
+  domainToCheck: string,
+  isValidDomain: boolean
+};
+type Props = {
+  resolveDomainRequested: (domain: string) => void
+};
+type HOCPassedComponent = any; //TODO: ComponentType (Property not found in exports of "react")
```

Comment:
`React.Component<*>` works

Reply:
I get an `[flow] type application of Component (Too few type arguments. Expected at least 3 See type parameters of definition here)` error from changing it to this